src/main/java/org/openstreetmap/atlas/utilities/configuration/StandardConfiguration.java

```diff
@@ -75,6 +78,7 @@ private StandardConfigurable(final String key, final Raw defaultValue,
         }
     }
 
+    private static final String OVERRIDE_STRING = "override";
```

Comment:
Maybe a quick note that this can no longer be used as a key. It says that in the PR description, might also help users in the code base.